docs/tasks/configure-pod-container/downward-api-volume-expose-pod-information.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Exposing Pod Information Using a DownwardApiVolumeFile
+---
+
+{% capture overview %}
+
+This page shows how a Pod can use a DownwardAPIVolume to expose information
+about itself to Containers running in the Pod. A DownwardAPIVolume can expose
+Pod fields and Container fields.
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+## Storing Pod fields
+
+In this exercise, you create a Pod that has one Container.
+Here is the configuration file for the Pod:
+
+{% include code.html language="yaml" file="dapi-volume.yaml" ghlink="/docs/tasks/configure-pod-container/dapi-volume.yaml" %}
+
+In the configuration file, you can see that the Pod has a `downwardAPI` Volume,
+and the Container mounts the Volume at `/etc`.
+
+Look at the `items` array under `downwardAPI`. Each element of the array is a
+[DownwardAPIVolumeFile](/docs/resources-reference/v1.5/#downwardapivolumefile-v1).
+The first element specifies that the value of the Pod's
+`metadata.labels` field should be stored in a file named `labels`.
+The second element specifies that the value of the Pod's `annotations` 
+field should be stored in a file named `annotations`.
+
+**Note**: The fields in this example are Pod fields. They are not
+fields of the Container in the Pod.
+
+Create the Pod:
+
+```shell
+kubectl create -f http://k8s.io/docs/tasks/configure-pod-container/dapi-volume.yaml
+```
+
+Verify that Container in the Pod is running:
+
+```shell
+kubectl get pods
+```
+
+View the Container's logs:
+
+```shell
+kubectl logs kubernetes-downwardapi-volume-example
+```
+
+The output shows the contents of the `labels` file and the `annotations` file:
+
+```shell
+cluster="test-cluster1"
+rack="rack-22"
+zone="us-est-coast"
+
+build="two"
+builder="john-doe"
+```
+
+Get a shell into the Container that is running in your Pod:
+
+```
+kubectl exec -it kubernetes-downwardapi-volume-example -- sh
+```
+
+In your shell, view the `labels` file:
+
+```shell
+/# cat /etc/labels
+```
+
+The output shows that all of the Pod's labels have been written
+to the `labels` file:
+
+```shell
+cluster="test-cluster1"
+rack="rack-22"
+zone="us-est-coast"
+```
+
+Similarly, view the `annotations` file:
+
+```shell
+/# cat /etc/annotations
+```
+
+View the files in the `/etc` directory:
+
+```shell
+/# ls -laR /etc
+```
+
+In the output, you can see that the `labels` and `annotations` files
+are in a temporary subdirectory: in this example,
+`..2982_06_02_21_47_53.299460680' In the `/etc` directory, `..data` is
```

Comment:
Typo: closing mark should be back tick instead of single quote, i.e.
`..2982_06_02_21_47_53.299460680' -> `..2982_06_02_21_47_53.299460680`